nuxeo-features/nuxeo-automation/nuxeo-automation-server/src/main/java/org/nuxeo/ecm/automation/server/jaxrs/batch/BatchManagerConstants.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * (C) Copyright 2016 Nuxeo SA (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Gabriel Barata <gbarata@nuxeo.com>
+ *
+ */
+package org.nuxeo.ecm.automation.server.jaxrs.batch;
+
+/**
+ * @since 8.4
+ */
+public class BatchManagerConstants {
+
+    public static final String NO_DROP_FLAG = "nx_batch_no_drop";
```

Comment:
The initial point of the X- prefix was to be compliant with the IETF recommendation on custom header.
It seems that since 2012, they have changed it and now using X- prefix is discouraged.

For consistency reason, as Antoine said, we have to use X- for now, but we could create an other task to get rid of all the X- headers and replace with Nuxeo- headers ...